src/main/java/com/project/Investment/App/config/AppBeanConfig.java

```diff
@@ -19,4 +23,24 @@ public JdbcTemplate jdbcTemplate(DataSource dataSource) {
     public Runner runner() {
         return new Runner();
     }
+
+    @Bean
+    public Connection connection() {
+        String URL = "jdbc:h2:mem:investment";
+        String USERNAME = "sa";
+        String PASSWORD = "";
+        try {
+            Class.forName("org.h2.Driver");
+        } catch (ClassNotFoundException sqlException) {
+            sqlException.printStackTrace();
+        }
+        try {
+            Connection  connection = DriverManager.getConnection( URL, USERNAME, PASSWORD);
+            log.info("Connect");
```

Comment:
Your log message should be somehow useful to a reader. This one doesn't tell anything

Reply:
Done